<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle"><string>Modern C++ Demo Launcher</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QComboBox" name="demoSelector"/>
    </item>
    <item>
     <widget class="QGroupBox" name="tagGroupBox">
      <property name="title">
       <string>Filter by Tag</string>
      </property>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QCheckBox" name="tagCommon">
         <property name="text">
          <string>Common</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tagDesignPatterns">
         <property name="text">
          <string>DesignPatterns</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tagCPP17">
         <property name="text">
          <string>CPP17</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tagCPP20">
         <property name="text">
          <string>CPP20</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tagNeuralNetwork">
         <property name="text">
          <string>NeuralNetwork</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text"><string>Run Selected Demo</string></property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="terminalOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
